apps/studio/src/components/editor/ResultTable.vue

```diff
@@ -304,6 +311,19 @@
             )
           )
         }
+      },
+      // HACK (day): this is probably not the best way of doing things, but postgres intervals are dumb
+      intervalSorter(a, b, aRow, bRow, column, dir, sorterParams) {
+        try {
+          const durationA = td.parse(intervalParse(a).toISOString());
+          const durationB = td.parse(intervalParse(b).toISOString());
+          const dateA = new Date(durationA.years, durationA.months, durationA.days, durationA.hours, durationA.minutes, durationA.seconds);
+          const dateB = new Date(durationB.years, durationB.months, durationB.days, durationB.hours, durationB.minutes, durationB.seconds);
+          return dateA - dateB;
+        } catch {
+          console.error('ERROR SORTING INTERVALS');
```

Comment:
So if this errors it could create O(n*n) console errors right? Maybe just have it return 0 without doing a console error. Not ideal but console spam can really slow the app.